cmd/hyperkube/main.go

```diff
@@ -46,5 +46,11 @@ func main() {
 	hk.AddServer(NewKubeProxy())
 	hk.AddServer(NewKubeAggregator())
 
+	// Alpha servers
+	hk.AddAlphaServer(NewCloudControllerManager())
+
+	// Alpha servers
+	hk.AddAlphaServer(NewCloudControllerManager())
```

Comment:
Should this be here twice?

Reply:
No, it shouldn't. Maybe a rebase error? Fixed in a new commit. (Will squash)